EncoreTickets.ConsoleTester/Program.cs

```diff
@@ -14,24 +14,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            ApiContext context = new ApiContext(Environments.Sandbox);
-            List<string> productIds = new List<string>();
-            context.affiliate = "encoretickets";
-            // uscontext.useBroadway = true;
+            var context = new ApiContext(Environments.Sandbox, "admin", "password") {affiliate = "encoretickets"};
+            // context.useBroadway = true;
+            var productIds = new List<string>();
+
+            Console.WriteLine();
+            Console.WriteLine(" ========================================================== ");
+            Console.WriteLine(" Test: Get JWT token for the venue service");
+            Console.WriteLine(" ========================================================== ");
+            var vsAuthApi = new VenueAuthenticationServiceApi(context);
+            context = vsAuthApi.Authenticate();
+
+            Console.WriteLine($"username: {context.userName}");
+            Console.WriteLine($"password: {context.password}");
+            Console.WriteLine($"token: {context.accessToken}");
 
             Console.WriteLine();
             Console.WriteLine(" ========================================================== ");
             Console.WriteLine(" Test: Get standard attributes ");
             Console.WriteLine(" ========================================================== ");
-            VenueServiceApi vsApi = new VenueServiceApi(context);
-            IList<EncoreTickets.SDK.Venue.StandardAttribute> stas = vsApi.GetStandardAttributes();
+            var vsApi = new VenueServiceApi(context);
```

Comment:
why change this? is it bad to type hint the declaration?

Reply:
If you prefer to use a specific type in the declaration, I can undo it. As I know, using var is a common practice in C #, and I think that it helps to abstract from a certain type, which improves reading code and speeds up updating code in the future.

Reply:
I am just saying cos I come from a java background. if this is a better convention in c#, I am ok with it